cli/pkg/kctrl/cmd/package/installed/status.go

```diff
@@ -61,6 +61,9 @@ func NewStatusCmd(o *StatusOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 
 func (o *StatusOptions) Run(args []string) error {
 	if o.pkgCmdTreeOpts.PositionalArgs {
+		if len(args) == 0 {
+			return fmt.Errorf("Package name not provided.")
+		}
 		o.Name = args[0]
```

Comment:
Good catch! 
Since we are already checking for the name on the next line, we could probably just do:
```suggestion
		if len(args) > 0 {
			o.Name = args[0]
		}
```

Reply:
I am also fine with that approach. I usually find it more readable to fail fast and avoid `else` clauses.

With the suggested approach we would need an else block.

Reply:
Applied! PTAL.